<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialog</class>
 <widget class="QDialog" name="AddItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1" columnstretch="1,1,1,1">
   <item row="3" column="0" colspan="4">
    <widget class="QListView" name="pointListView"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="yLabel">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPushButton" name="addPointPushButton">
     <property name="text">
      <string>Add Point</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="yLineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="xLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>X:</string>
     </property>
     <property name="buddy">
      <cstring>xLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>xLineEdit</tabstop>
  <tabstop>yLineEdit</tabstop>
  <tabstop>addPointPushButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>pointListView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
